src/field_weather_effect.c

```diff
@@ -1021,17 +1021,17 @@ enum {
 
     // The thunderstorm loops through these states,
     // not necessarily in order.
-    TSTORM_STATE_LOOP_START,
-    TSTORM_STATE_LOOP_WAIT,
-    TSTORM_STATE_INIT_THUNDER_SHORT_1,
-    TSTORM_STATE_INIT_THUNDER_SHORT_2,
-    TSTORM_STATE_TRY_THUNDER_SHORT,
-    TSTORM_STATE_TRY_NEW_THUNDER,
-    TSTORM_STATE_WAIT_THUNDER_SHORT,
-    TSTORM_STATE_INIT_THUNDER_LONG,
-    TSTORM_STATE_WAIT_THUNDER_LONG,
-    TSTORM_STATE_FADE_THUNDER_LONG,
-    TSTORM_STATE_END_THUNDER_LONG,
+    TSTORM_STATE_NEW_CYCLE,
+    TSTORM_STATE_NEW_CYCLE_WAIT,
+    TSTORM_STATE_INIT_CYCLE_1,
+    TSTORM_STATE_INIT_CYCLE_2,
+    TSTORM_STATE_SHORT_BOLT,
+    TSTORM_STATE_TRY_NEW_BOLT,
+    TSTORM_STATE_WAIT_BOLT_SHORT,
+    TSTORM_STATE_INIT_BOLT_LONG,
+    TSTORM_STATE_WAIT_BOLT_LONG,
+    TSTORM_STATE_FADE_BOLT_LONG,
+    TSTORM_STATE_END_BOLT_LONG,
```

Comment:
I get what you mean, but `TSTORM` looks like a spelling mistake; would it be worthwhile to use either `STORM` or `THUNDERSTORM`?

Reply:
Switched to `THUNDER`